### PR TITLE
Update productDetail.js

### PR DESCRIPTION
**Pull request by LaNerva**
Star section adjusted, from plain text to productRatingAverage

### CODE REVIEW

File: src/components/productDetail.js [hunk 1]
```diff
@@ -26,7 +26,7 @@ function ProductDetail({ productInformation }) {
         </div>
         <h2 className="product-title">{productInformation.name}</h2>
         <div className="star-section">
-          <p>★★★★☆</p>
+          <p>{productInformation.ratingAverage}</p>
```
Comment (PedroDCA) [inline src/components/productDetail.js:29]: Here are we just showing the number?
The original idea is to show the same starts than the rating average, where if the product has a 4 rating average, it should show 4 filled stars and just one not filled, and so on.

**Review by PedroDCA — CHANGES_REQUESTED** [review]

File: src/components/star.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+function Star({ filled }) {
+    return <span className={filled ? "star filled" : "star"}>{filled ? '★' : '☆'}</span>;
```
Comment (PedroDCA) [inline src/components/star.js:2]: Not a fan about using special characters on HTML as it could break HTML according to the computer if the character does not exist.
Please use only css to be able to show the filled start and the non-filled star

**Review by PedroDCA — CHANGES_REQUESTED** [review]

File: src/components/productDetail.js [hunk 3]
```diff
@@ -11,6 +12,15 @@ function ProductDetail({ productInformation }) {
   const buttonText = productInformation.ableToAddCard
     ? "Agregar al carrito"
     : "En carrito";
+
+    const renderStars = () => {
+      let stars = [];
+      for (let i = 1; i <= 5; i++) {
+        stars.push(<Star key={i} filled={i <= productInformation.ratingAverage} />);
```
Comment (PedroDCA) [inline src/components/productDetail.js:19]: Let's try to keep this login on the return, in other words, we have to iterate some list by using map function and then return the Star component you created.
You could create a list from a number like [...Array(5)] and use map to render the stars, where on the map callback function you get the logic to know when it should be filled

**Review by PedroDCA — CHANGES_REQUESTED** [review]

File: package.json [hunk 4]
```diff
@@ -21,6 +21,7 @@
     "react-router-dom": "^6.18.0",
     "react-scripts": "5.0.1",
     "redux-persist": "^6.0.0",
+    "sweetalert": "^2.1.2",
```
Comment (PedroDCA) [inline package.json:24]: Why are you installing it?

File: src/styles/star.css
Comment (PedroDCA) [inline src/styles/star.css:1]: I think you are not using these styles

File: package.json (hunk 4 above)
Comment (LaNerva) [reply package.json:24]: The app was giving errors before installing sweetalert, once I did it started working.

Comment (PedroDCA) [reply package.json:24]: Please revert this changes (it applies the same for package-lock.json) y just run `npm install`, it should fix this issue. Otherwise, please share an screenshot about the issue

File: package-lock.json
Comment (PedroDCA) [inline package-lock.json:1]: Please remove the changes form here

File: src/styles/star.css
Comment (PedroDCA) [reply src/styles/star.css:1]: Please review this

Comment (LaNerva) [reply src/styles/star.css:1]: Yup, you are right. I was using those styles for previous version of the star rating function.

File: src/components/star.js [hunk 5]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+function Star({ filled, half }) {
+  const starPath = "M12 .587l3.688 7.462 8.312 1.207-6 5.854 1.416 8.263L12 19.897l-7.416 3.876 1.416-8.263-6-5.854 8.312-1.207Z";
+  const halfStarPath = "M9.5 14.25l-5.584 2.936 1.066-6.218L.465 6.564l6.243-.907L9.5 0l2.792"; 
+
+  return (
+    <svg className={`star ${filled ? 'filled' : ''} ${half ? 'half-filled' : ''}`} width="24" height="24" viewBox="0 0 24 24">
```
Comment (PedroDCA) [inline src/components/star.js:8]: Where are the css properties for the `filled `and `half-filled `classes?

**Review by PedroDCA — CHANGES_REQUESTED** [review]

File: src/components/productDetail.js [hunk 6]
```diff
@@ -2,31 +2,44 @@ import { useDispatch } from "react-redux";
 import { formatPriceForColonCurrency } from "../service/priceService";
 import { addCartElement } from "../store"
 import "../styles/productDetail.css"
+import Star from "./star";
 
 function ProductDetail({ productInformation }) {
+
+
+
```
Comment (PedroDCA) [inline src/components/productDetail.js:10]: Please remove those empty lines and fix the merge conflicts

File: package-lock.json
Comment (PedroDCA) [inline package-lock.json:1]: Please revert the changes from this file

**Review by PedroDCA — CHANGES_REQUESTED** [review]

File: src/components/productDetail.js (hunk 6 above)
Comment (LaNerva) [reply src/components/productDetail.js:10]: what do you mean by merge conflicts?

**Review by PedroDCA — APPROVED** [review]
